api/WorkerNavigator.json

```diff
@@ -401,6 +449,102 @@
           }
         }
       },
+      "locks": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/web-locks/#dom-navigatorlocks-locks",
+          "support": {
+            "chrome": {
+              "version_added": "69"
+            },
+            "chrome_android": {
+              "version_added": "69"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "56"
+            },
+            "opera_android": {
+              "version_added": "48"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": "10.0"
+            },
+            "webview_android": {
+              "version_added": "69"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "mediaCapabilities": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/media-capabilities/#dom-navigator-mediacapabilities",
+          "support": {
+            "chrome": {
+              "version_added": "76"
```

Comment:
Doesn't match api.Navigator.mediaCapabilities, but I've confirmed the difference is real, worker support came later, tracked by https://www.chromestatus.com/feature/5730672977117184.